Doc/c-api/unicode.rst

```diff
@@ -1396,18 +1396,28 @@ They all return ``NULL`` or ``-1`` if an exception occurs.
    :c:func:`PyErr_Occurred` to check for errors.
 
 
-.. c:function:: int PyUnicode_EqualToUTF8(PyObject *unicode, const char *string)
+.. c:function:: int PyUnicode_EqualToUTF8AndSize(PyObject *unicode, const char *string, Py_ssize_t size)
```

Comment:
What do you think about renaming `string` to `utf8_str`? The `utf8_` would be another way to document that it's expected to be encoded to UTF-8 and also it's easier (for me) to distinguish that the second argument is a bytes string, since `string` name is quite generic.

Reply:
It is a part of the bigger issue. See #62897.